File: Ex15.c
#include<stdio.h>


int main(int argc, char const *argv[])
{
    // Inserção de variáveis
    int Peso, IMC;
    float Altura;

    // Inserção de valor
    printf("Insira o Peso do paciente:");
    scanf("%i",&Peso);

    // Inserção de valor
    printf("Insira a Altura do paciente:");
    scanf("%f",&Altura);

    // Cálculo do IMC
    IMC= Peso/(Altura*Altura);

    // Comparação de variáveis

    if (IMC<20)
    {
        printf("Abaixo do Peso.");
    }
    
    else {
        if (IMC<25)
        {
            printf("Peso ideal.");
        }
        else {
            printf("Acima do Peso.");
        }
        
    }
    







    return 0;
}

File: Ex15.py
peso = float(input("Insira seu peso para calculo de IMC: "))
altura = float(input("Insira sua altura: "))

imc= (peso/(altura*altura))

print(f"O seu IMC e de {imc:.2f}.")

if imc < 20:
    print("Voce esta abaixo do peso ideal.")

elif imc >= 20 and imc <= 25:
    print("Voce esta no peso ideal.")

else:
    print("Voce esta acima do peso.")